src/Solana/Hash.h

```diff
@@ -0,0 +1,27 @@
+// Copyright © 2017-2023 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include <array>
+#include <string>
+#include "Base58.h"
+
+namespace TW::Solana {
+    class Hash {
```

Comment:
I know this is old code, it's not `Hash` tho, just base58 encoding / decoding IMHO, since you're refactoring here